### ISSUE TITLE
craft rebuild console command looks regressed

### ISSUE THREAD

**Issue by narration-sd** [craftcms/cms#4962]
### Description

Just following directions for starting up project config, for a test, the rebuild command failed, in a way I think we'd recognize and well understand...

### Steps to reproduce

1. here's the command  exec and stack trace result:
```
vagrant@lv-ngatsby:~/lv-ngatsby$ ./craft project-config/rebuild


Exception 'yii\base\UnknownMethodException' with message 'Calling unknown method: craft\console\Request::getReferrer()'

in /home/vagrant/lv-ngatsby/vendor/yiisoft/yii2/base/Component.php:300

Stack trace:
#0 /home/vagrant/lv-ngatsby/vendor/narrationsd/live-vue/src/helpers/RouteHelpers.php(46): yii\base\Component->__call('getReferrer', Array)
#1 /home/vagrant/lv-ngatsby/vendor/narrationsd/live-vue/src/LiveVue.php(149): narrationsd\livevue\helpers\RouteHelpers->__construct(true, true)
#2 /home/vagrant/lv-ngatsby/vendor/yiisoft/yii2/base/BaseObject.php(109): narrationsd\livevue\LiveVue->init()
#3 /home/vagrant/lv-ngatsby/vendor/yiisoft/yii2/base/Module.php(158): yii\base\BaseObject->__construct(Array)
#4 /home/vagrant/lv-ngatsby/vendor/craftcms/cms/src/base/Plugin.php(127): yii\base\Module->__construct('live-vue', Object(craft\console\Application), Array)
#5 [internal function]: craft\base\Plugin->__construct('live-vue', Object(craft\console\Application), Array)
#6 /home/vagrant/lv-ngatsby/vendor/yiisoft/yii2/di/Container.php(384): ReflectionClass->newInstanceArgs(Array)
#7 /home/vagrant/lv-ngatsby/vendor/yiisoft/yii2/di/Container.php(156): yii\di\Container->build('narrationsd\\liv...', Array, Array)
#8 /home/vagrant/lv-ngatsby/vendor/yiisoft/yii2/BaseYii.php(349): yii\di\Container->get('narrationsd\\liv...', Array, Array)
#9 /home/vagrant/lv-ngatsby/vendor/craftcms/cms/src/services/Plugins.php(897): yii\BaseYii::createObject(Array, Array)
#10 /home/vagrant/lv-ngatsby/vendor/craftcms/cms/src/services/Plugins.php(230): craft\services\Plugins->createPlugin('live-vue', Array)
#11 /home/vagrant/lv-ngatsby/vendor/craftcms/cms/src/base/ApplicationTrait.php(1334): craft\services\Plugins->loadPlugins()
#12 /home/vagrant/lv-ngatsby/vendor/craftcms/cms/src/console/Application.php(65): craft\console\Application->_postInit()
#13 /home/vagrant/lv-ngatsby/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\console\Application->init()
#14 /home/vagrant/lv-ngatsby/vendor/yiisoft/yii2/base/Application.php(206): yii\base\BaseObject->__construct(Array)
#15 /home/vagrant/lv-ngatsby/vendor/yiisoft/yii2/console/Application.php(89): yii\base\Application->__construct(Array)
#16 /home/vagrant/lv-ngatsby/vendor/craftcms/cms/src/console/Application.php(54): yii\console\Application->__construct(Array)
#17 [internal function]: craft\console\Application->__construct(Array)
#18 /home/vagrant/lv-ngatsby/vendor/yiisoft/yii2/di/Container.php(384): ReflectionClass->newInstanceArgs(Array)
#19 /home/vagrant/lv-ngatsby/vendor/yiisoft/yii2/di/Container.php(156): yii\di\Container->build('craft\\console\\A...', Array, Array)
#20 /home/vagrant/lv-ngatsby/vendor/yiisoft/yii2/BaseYii.php(349): yii\di\Container->get('craft\\console\\A...', Array, Array)
#21 /home/vagrant/lv-ngatsby/vendor/craftcms/cms/bootstrap/bootstrap.php(248): yii\BaseYii::createObject(Array)
#22 /home/vagrant/lv-ngatsby/vendor/craftcms/cms/bootstrap/console.php(34): require('/home/vagrant/l...')
#23 /home/vagrant/lv-ngatsby/craft(21): require('/home/vagrant/l...')
#24 {main}
```
2.

### Additional info

PHP version	7.1.28
OS version	Linux 4.15.0-47-generic
Database driver & version	MySQL 5.7.25
Image driver & version	GD 7.1.28
Craft edition & version	Craft Pro 3.3.4.1
Yii version	2.0.21
Twig version	2.11.3
Guzzle version	6.3.3
Imagine version	1.2.3-dev

**Comment by andris-sevcenko** [comment]
But it looks like it's coming from your plugin?

Project config needs to initialize every installed plugin when rebuild is run to see if plugins have anything to contribute there.

**Comment by narration-sd** [comment]
Ah, right. That looks pretty dumb, doesn't it. Because I just 'knew' what _that_ problem was....never even glanced at what I pasted.

The underlying factor though. points probably to documentation. I was in fact reading that and being told that I didn't have any reason to do anything (strong suggestion not to) with this plugin for Project Config. But I do...just not the indicated depth steps.

And of course this was just the supposedly preceding step to turning on PC, which I then did for a first, and did look quite carefully at what came out of that. Not so bad.

I'll fix this, and maybe come up with some language that could be put in for the future unwary. Ahead of all that which is intricate. And of course all of this was to assure that plugin got along with Project Config. So, a success :)
